locale/00_UpdateLocalization.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../RubySetupSystem/RubyCommon'
```

Comment:
I think this script should be put in the scripts folder and called something like `update_localization.rb` to match with naming convention.